Source/Atomic/UI/UIWidget.h

```diff
@@ -250,10 +258,18 @@ class UIWidget : public Object, public tb::TBWidgetDelegate
     bool GetCaptured();
 
     void SetCapturing(bool capturing);
```

Comment:
I should have flagged these before, GetCapturing/SetCapturing are unclear as to what they do.  Can you describe them a bit, and they we can work on what the names should be?